src/Core/Components/NavMenu/FluentNavMenu.razor.js

```diff
@@ -1,37 +1,47 @@
 export function onLoad() {
-    const mql = window.matchMedia("(max-width: 600px)");
-
-    for (let expander of document.getElementsByClassName("expander")) {
-        if (expander) {
-            const origStyle = expander.parentElement.style.cssText;
-            expander.addEventListener('click', (ev) => toggleMenuExpandedAsync(expander, origStyle, ev));
-            expander.addEventListener('keydown', (ev) => handleMenuExpanderKeyDownAsync(expander, origStyle, ev));
-
-            mql.onchange = (e) => {
-                if (e.matches) {
-                    setMenuExpanded(expander, origStyle, true)
-                }
-            };
+    if (!isRunningInWASM()) {
+        const mql = window.matchMedia("(max-width: 600px)");
+
+        for (let expander of document.getElementsByClassName("expander")) {
+            if (expander) {
+                const origStyle = expander.parentElement.style.cssText;
+                expander.addEventListener('click', (ev) => toggleMenuExpandedAsync(expander, origStyle, ev));
+                expander.addEventListener('keydown', (ev) => handleMenuExpanderKeyDownAsync(expander, origStyle, ev));
+
+                mql.onchange = (e) => {
+                    if (e.matches) {
+                        setMenuExpanded(expander, origStyle, true)
+                    }
+                };
+            }
+        }
+        for (let element of document.getElementsByClassName("fluent-nav-group")) {
+            attachEventHandlers(element);
         }
-    }
-    for (let element of document.getElementsByClassName("fluent-nav-group")) {
-        attachEventHandlers(element);
     }
 }
+
 export function onUpdate() {
-    
+
 }
 
 export function onDispose() {
-    for (let expander of document.getElementsByClassName("expander")) {
-        if (expander) {
-            expander.removeEventListener('click', toggleMenuExpandedAsync);
-            expander.removeEventListener('keydown', handleMenuExpanderKeyDownAsync);
+    if (!isRunningInWASM()) {
+        for (let expander of document.getElementsByClassName("expander")) {
+            if (expander) {
+                expander.removeEventListener('click', toggleMenuExpandedAsync);
+                expander.removeEventListener('keydown', handleMenuExpanderKeyDownAsync);
+            }
+        }
+        for (let element of document.getElementsByClassName("fluent-nav-group")) {
+            detachEventHandlers(element);
         }
     }
-    for (let element of document.getElementsByClassName("fluent-nav-group")) {
-        detachEventHandlers(element);
-    }
+}
+
+function isRunningInWASM() {
```

Comment:
This test is dependent on the implementation of the **DotNet** in .net 8.

Reply:
Where does this command come from? Do you have a reference in the documentation?

What happens in "auto" mode where the component is initially displayed with interactive server-side rendering (interactive SSR) using the Blazor Server hosting model. The .NET WASM runtime and the entire application are downloaded to the client in the background and cached for use on subsequent visits.